CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Added validation of versions in the cloud config Params struct. Versions outside of supported ranges will cause an
   error to be returned from cloud config-related functions.
+  
+### Changed
+
+- Update Kubernetes dependencies to v1.18.5.
```

Comment:
```suggestion
- Updated backward incompatible Kubernetes dependencies to v1.18.5.
```

k8scloudconfig is another library lots of people touch. So can we go with the same as Pawel suggested in operatorkit?

Reply:
Got it, Let's use the same sentence here.